### PR TITLE
 improve Status Chinese display and fix scheduling time display error

### PR DESCRIPTION
**Pull request by TianWuwt**
## Summary

1. improve Status Chinese display
Pasted Graphic 1
Pasted Graphic
Pasted Graphic 2
Pasted Graphic 3
Pasted Graphic 4
Pasted Graphic 5
Pasted Graphic 6
2. fix scheduling time display error

### CODE REVIEW

File: ui/src/constants/index.ts [hunk 1]
```diff
@@ -379,3 +367,109 @@ export {
   ZONE_IMG_MAP,
   getMinResource,
 };
+
+export const STATUS_LIST = [
```
Comment (powerfooI) [inline ui/src/constants/index.ts:371]: Why no use map directly instead of list if all the usages are paired with `findByValue`?

Comment (TianWuwt) [reply ui/src/constants/index.ts:371]: The data processed by the map cannot be used directly and is prone to errors.

Comment (powerfooI) [reply ui/src/constants/index.ts:371]: I think `{"running": {...}}` can be used directly. What if `findByValue` find nothing if you use list? When talking about error-proneness lists are almost the same as maps.

Comment (TianWuwt) [reply ui/src/constants/index.ts:371]: Different methods, different habits, the same results （＾∀＾）

**Review by powerfooI — APPROVED** [review]
LGTM

**Pull request merged**